docs/index.md

```diff
@@ -106,11 +106,12 @@ We gratefully acknowledge the following projects for supporting the development
 
 
 [Tutorial]: https://emmc-asbl.github.io/tripper/latest/tutorial/
+[Documentation]: https://emmc-asbl.github.io/tripper/latest/
```

Comment:
Documentation now points to just the whole documentation. This is strange to have under "Getting started". I suppose this should be about how to do data documentation? It needs a clearer name and correct link.

Reply:
As discussed, this comes from the main readme file and is the "Getting started" for tripper itself.